### PR TITLE
Fix references to state.document.notebook

### PR DESCRIPTION
**Pull request by captainsafia**
Immutable.js objects don't play nice with ES6 destructuring.

### CODE REVIEW

File: src/notebook/menu/index.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ export function triggerSaveAs(store, dispatch) {
         return;
       }
       const state = store.getState();
-      const { notebook, executionState } = state.document;
+      const { executionState } = state.document;
```
Comment (jdfreder) [inline src/notebook/menu/index.js:39]: Here you use `state.document.executionState` but above you use `state.document.get` to access notebook.  Is `state.document` immutable or not?

Comment (jdfreder) [inline src/notebook/menu/index.js:39]: IOW, I think one of the lines is wrong

Comment (jdfreder) [inline src/notebook/menu/index.js:39]: Ah, looks like you are using `get` everywhere.  This line should be a `get` call too.  If `get` causes it to break, that means the code setting the execution state on the state.document is doing it wrong

Comment (rgbkrk) [inline src/notebook/menu/index.js:39]: Yeah this needs to be extracted as well using `.get`

**Comment by rgbkrk** [comment]
These are all things I missed during the Immutable refactor because they weren't tested (there's coverage, though it doesn't appear to be checking the logic).

File: src/notebook/menu/index.js (hunk 1 above)
Comment (captainsafia) [inline src/notebook/menu/index.js:39]: Whoops! Yeah, I missed this one. No PRs after midnight for me!

**Comment by captainsafia** [comment]
I'll open another PR later today with fixes and do a more thorough pass for this in the code.

**Pull request merged**